junifer/configs/juseless/datagrabbers/ixi_vbm.py

```diff
@@ -0,0 +1,91 @@
+"""Provide class for juseless datalad datagrabber."""
+
+# Authors: Federico Raimondo <f.raimondo@fz-juelich.de>
+#          Leonard Sasse <l.sasse@fz-juelich.de>
+#          Synchon Mandal <s.mandal@fz-juelich.de>
+# License: AGPL
+
+from pathlib import Path
+from typing import List, Union
+
+from ....api.decorators import register_datagrabber
+from ....datagrabber import PatternDataladDataGrabber
+from ....utils import raise_error
+
+
+@register_datagrabber
+class JuselessDataladIXIVBM(PatternDataladDataGrabber):
+    """Juseless IXI VBM DataGrabber class.
+
+    Implements a DataGrabber to access the IXI VBM data in Juseless.
+
+    Parameters
+    -----------
+    datadir : str or pathlib.Path, optional
+        The directory where the datalad dataset will be cloned. If None,
+        the datalad dataset will be cloned into a temporary directory
+        (default None).
+    sites : {"Guys", "HH", "IOP"} or list of the
+        options, optional.
+        Which sites to access data from. If None, all available sites are
+        selected (default None).
+    """
+
+    def __init__(
+        self,
+        datadir: Union[str, Path, None] = None,
+        sites: Union[str, List[str], None] = None,
+    ) -> None:
+        """Initialize the class."""
+        uri = "git@jugit.fz-juelich.de:inm7/datasets/datasets_repo.git"
+        types = ["VBM_GM"]
+        rootdir = "processed/cat_12.5/IXI"
+        replacements = ["site", "subject"]
+        patterns = {
+            "VBM_GM": "{site}/sub-{subject}/mri/m0wp1sub-{subject}.nii.gz"
+        }
+
+        # validate and/or transform 'site' input
+        all_sites = "HH", "Guys", "IOP"
```

Comment:
I think it should be: `all_sites = ["HH", "Guys", "IOP"]`

Reply:
oops yes